src/core/File.ts

```diff
@@ -117,6 +117,11 @@ export class File {
 
     public groupHandler: Plugin;
 
+    /**
+     * Flag to indicate to fusebox whether or not contents has been set to consumable javascript
+     */
+    public transpiled: Boolean = false;
```

Comment:
I would rename this one to `isTypescriptTranspiled` or maybe some better name, can't just figure out what would that be.  cuz `transpiled` is quite a generic name, we do however a first class typescript support. 

Could you create some documentation in the plugin section to help people understand and make use of this option?